README.md

```diff
@@ -79,7 +79,8 @@ end
 | scope                        | Which OpenID scopes to include (:openid is always required)                                                                                                              | no       | Array<sym> [:openid]          | [:openid, :profile, :email]                         |
 | response_type                | Which OAuth2 response type to use with the authorization request                                                                                                         | no       | String: code                  | one of: 'code', 'id_token'                          |
 | state                        | A value to be used for the OAuth2 state parameter on the authorization request. Can be a proc that generates a string.                                                   | no       | Random 16 character string    | Proc.new { SecureRandom.hex(32) }                   |
-| require_state                | Should state param be verified - this is recommended, not required by the OIDC specification                                                                             | no       | true                          | false                                               |
+| require_state                | Should the callback phase require that a state is present. If `sent_state` is true, then the callback state must match the authorize state. This is recommended, not required by the OIDC specification. | no       | true                          | false                                               |
```

Comment:
```suggestion
| require_state                | Should the callback phase require that a state is present. If `send_state` is true, then the callback state must match the authorize state. This is recommended, not required by the OIDC specification. | no       | true                          | false                                               |
```

Reply:
As follow-up we can maybe add a column with OIDC specification and add optional/required to all the options